packages/core/src/i18n/translations/tl.js

```diff
@@ -1,7 +1,8 @@
 export default {
   'error': 'Error',
   'required': '(*Kailangan)',
-  'max-chars': '(Pinakamarami ang {{length}} character)',
+  'max-chars': '(Pinakamarami ang {{length}} karakter)',
+  'min-chars': '(Pinakamababa ang {{length}}) karakter)',
```

Comment:
Is this right for min-chars? It seems like there's an extra `)` in there next to the number but maybe that's what the translation told us to do?

![Screenshot 2023-01-13 at 9 51 31 AM](https://user-images.githubusercontent.com/872479/212361865-3a6a9581-c163-417e-b0b5-ddc4ce892a62.png)

The max length one looks like this:
![Screenshot 2023-01-13 at 9 54 18 AM](https://user-images.githubusercontent.com/872479/212362434-99f58380-e6d0-42d1-81e8-060fb965e0a0.png)

Reply:
Good catch!